data/tlg0007/tlg068/tlg0007.tlg068.perseus-grc2.xml

```diff
@@ -0,0 +1,2231 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="lat">Quomodo adolescens poetas audire debeat</title>
+<!--<title type="sub">Machine readable text</title>-->
+<author>Plutarch</author>
+<editor role="editor" n="Teubner">Gregorius N.
+            Bernardakis</editor>
+<sponsor>Perseus Project, Tufts University</sponsor>
+<principal>Gregory Crane</principal>
+<respStmt>
+<resp>Prepared under the supervision of</resp>
+<name>Lisa Cerrato</name>
+<name>William Merrill</name>
+<name>Elli Mylonas</name>
+<name>David Smith</name>
+</respStmt>
+<funder n="org:NEH">The National Endowment for the Humanities</funder>
+</titleStmt>
+<!--<extent/>-->
+<publicationStmt>
+<publisher>Trustees of Tufts University</publisher>
+<pubPlace>Medford, MA</pubPlace>
+<authority>Perseus Project</authority>
+<idno type="filename">tlg0007.tlg068.perseus-grc2.xml</idno>
```

Comment:
We do not typically use this tag. Let's strike it.

Reply:
Please add `<date type="release">2010-12-13'</date>`. This is something that saves me a new PR. Thanks.